src/plotman/interactive.py

```diff
@@ -115,10 +116,16 @@ def curses_main(stdscr):
                     cfg.directories, cfg.scheduling, cfg.plotting
                 )
                 if (started):
+                    if aging_reason is not None:
+                        log.log(aging_reason)
+                        aging_reason = None
                     log.log(msg)
                     plotting_status = '<just started job>'
                     jobs = Job.get_running_jobs(cfg.directories.log, cached_jobs=jobs)
                 else:
+                    # If a plot is delayed for any reason other than stagger, log it
+                    if msg.find("stagger") < 0:
+                        aging_reason = msg
```

Comment:
So ... when we start a new plot job, we'll log the reason it had been delayed?  That could be useful.

Reply:
It has helped me with tuning my machines by suggesting what resources are lacking.  The only downside that I have encountered is that the Log: count message no longer reflects the total number of jobs launched by plotman since delay messages are also counted.